package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "webcomponents.org",
+  "name": "@webcomponents/internal-site",
```

Comment:
There is probably no feasible attack unless we publish, yeah. However, I think it's just a good habit to get into for npm development, because in other scenarios there are attacks. I think  we should simply never declare a package name that we don't own, or in a namespace that we don't own, as a rule.